lib/server.coffee

```diff
@@ -126,7 +124,7 @@ WebApp.connectHandlers.use (req, res, next) ->
 
 		Spiderable.originalRequest = req
 		url 		= Spiderable._urlForPhantom Meteor.absoluteUrl(), req.url
-		hash 		= SHA256 url
+		hash 		= hashCode url
```

Comment:
I'd be much more in favor of base64 primarily because it is standard. Since it is a simple character encoding, and possibly even optimized, it should be sufficiently fast. (Speed is not likely to be a bottleneck anyway).
It also guarantees 1:1 uniqueness.

(I am still not sure that special symbols would cause an issue.) In any case, I vote for base64 as well...